curve25519/curve25519Generate_test.go

```diff
@@ -0,0 +1,20 @@
+package curve25519_test
+
+import (
+	"github.com/TankerHQ/identity-go/curve25519"
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("curve25519Generate", func() {
+	It("generates a non nil key pair", func() {
+		sk, pk, err := curve25519.GenerateKey()
+		Expect(err).ShouldNot(HaveOccurred())
+		Expect(len(sk)).Should(Equal(32))
+		Expect(len(pk)).Should(Equal(32))
+
+		emptyArray := [32]byte{}
+		Expect(sk).ShouldNot(Equal(emptyArray[:]))
+		Expect(pk).ShouldNot(Equal(emptyArray[:]))
+	})
```

Comment:
You could test that if you generate two keys, they're actually different, just as a sanity check!

Reply:
Done